models/issue_comment.go

```diff
@@ -705,11 +705,11 @@ func (c *Comment) LoadPushCommits() (err error) {
 		c.NewCommit = data.CommitIDs[1]
 	} else {
 		repoPath := c.Issue.Repo.RepoPath()
-		gitRepo, err := git.OpenRepository(repoPath)
+		gitRepo, closer, err := git.RepositoryFromContextOrOpen(ctx, repoPath)
```

Comment:
just wonder what's the difference between `OpenRepositoryCtx` and `RepositoryFromContextOrOpen` ?

Reply:
never mid :swan: